### ISSUE TITLE
Override of island limits

### ISSUE THREAD

**Issue by LoneDevilX** [Guillaume-Lebegue/IslandUpgrades#29]
#### Describe the bug

Island upgrade add ons override the permission of island limits 'bskyblock.island.limit.hopper.xx'
I have already disable the upgrade hopper limits in upgrade's config
but its override, restrict the permission i used in island limits

#### Steps to reproduce the behavior

none

#### Expected behavior

ISland upgrade and limits  should be separate, since I use UPGRADE only for border, so player can rank up to get higher hopper limits without changing the border
But it override the island limits permission although I rankup, the hopper limits ain't increase

#### Screenshots and videos (Optional)

https://user-images.githubusercontent.com/67851836/93301801-8fea4a80-f82b-11ea-9076-dddf72e1dc7a.png
https://user-images.githubusercontent.com/67851836/93301910-b7411780-f82b-11ea-8254-5e83ccf187ea.png
##### Output of `/bbox version` (Mandatory)

```
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Running PAPER 1.15.2.
[Tue, 29. Sep 2020 16:43:41 SGT INFO] BentoBox version: 1.15.0
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Database: JSON
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Loaded Game Worlds:
[Tue, 29. Sep 2020 16:43:41 SGT INFO] bskyblock_world (BSkyBlock): Overworld, Nether, The End
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Loaded Addons:
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Biomes 1.13.0 (ENABLED)
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Border 2.0.0 (ENABLED)
[Tue, 29. Sep 2020 16:43:41 SGT INFO] BSkyBlock 1.13.1 (ENABLED)
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Challenges 0.8.3 (ENABLED)
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Chat 1.0.4 (ENABLED)
[Tue, 29. Sep 2020 16:43:41 SGT INFO] DimensionalTrees 1.6.0 (ENABLED)
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Level 2.4.1 (ENABLED)
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Limits 1.14.0 (ENABLED)
[Tue, 29. Sep 2020 16:43:41 SGT INFO] MagicCobblestoneGenerator 2.1.0 (ENABLED)
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Upgrades 0.2.0-SNAPSHOT-b64 (ENABLED)
[Tue, 29. Sep 2020 16:43:41 SGT INFO] Warps 1.9.7 (ENABLED)
```

#### Plugins (Optional)

```
_______
```

**Comment by LoneDevilX** [comment]
I have already submitted ticket about this before. Now i found out its the 'ISLAND UPGRADE' cause this
https://github.com/BentoBoxWorld/Limits/issues/101

Here is my config.
[Upgrades.zip](https://github.com/BentoBoxWorld/Upgrades/files/5297629/Upgrades.zip)

**Comment by Guillaume-Lebegue** [comment]
I made some things that make it possible to use upgrade and limits permission at the same times.

Only permission about limit that have upgrades will be ignored.

However, be careful as there is a bug from limits addon that reset upgrades on connection

**Comment by LoneDevilX** [comment]
I am trying to disable the hopper upgrade in upgrade config.yml
cause it conflicts with the limits permission.

somehow I execute rank up, it's still not fixed.

**Comment by Guillaume-Lebegue** [comment]
Yes, That's intended for now as the fix that I have made recreated the bug that reset limits on player log in.

I'm waiting for this issue https://github.com/BentoBoxWorld/Limits/issues/113

**Comment by LoneDevilX** [comment]
May I know how is it going now?

**Comment by Guillaume-Lebegue** [comment]
It's on pause for now. As to make this possible, I need to finish my rework of the settings

**Comment by LoneDevilX** [comment]
Sure. I will follow up the latest update.

**Comment by Guillaume-Lebegue** [comment]
See #37 for the advancement.
I will try to work on it for the next few days.
